components/automate-ui/src/app/pages/service-groups/service-groups.component.ts

```diff
@@ -200,16 +304,82 @@ export class ServiceGroupsComponent implements OnInit, OnDestroy {
     const sortField = this.getSelectedSortField(allParameters);
     const pageField = this.getSelectedPageNumber(allParameters);
     // Here we can add all the filters that the search bar will have
+    const searchBarFilters = fpFilter(chicklet => {
+      return some({'type': chicklet.type}, this.categoryTypes);
+    }, allParameters);
+
     const serviceGroupFilters: ServiceGroupFilters = {
       status: status,
       sortField: sortField,
       sortDirection: sortDirection,
       page: pageField,
-      pageSize: this.pageSize
+      pageSize: this.pageSize,
+      searchBar: searchBarFilters
     };
     this.store.dispatch(new UpdateServiceGroupFilters({filters: serviceGroupFilters}));
   }
 
+  get shareUrl() {
```

Comment:
```suggestion
  get shareUrl(): string {
```

And while you are in this file, if you don't mind, most of the functions in this file are in need of a `: void` showing no return type. Just a few functions here have it.